attestation.md

```diff
@@ -0,0 +1,89 @@
+# 🔖 Attestation 
+
+In this section, you'll play the role of an <span class="label-role attester">attester</span>:
+* You'll take a `RequestForAttestation` object; 
+* Attest it;
+* Store the attestation on the chain;
+* Build the `AttestedClaim` object, which will be send back to the <span class="label-role claimer">claimer</span>.
+
+## Get some tokens as an attester  
+
+You'll need some token in  order to attest a claim.   
+Go to https://faucet.kilt.io/ and request tokens for your `[attester address]`.
+
+## Code: create file 
+
+Create a new file `3-attestation.js`.
+All the following code needs to go into this file.  
+
+## Code: validate the `RequestForAttestation` object 
+
+In a real system: as an <span class="label-role attester">attester</span>, you would directly receive a `RequestForAttestation` from a  <span class="label-role claimer">claimer</span> via KILT's 1:1 messaging system.  
+
+In this tutorial, you can either:
+* take the `RequestForAttestation` object you've generated in the previous step as a <span class="label-role claimer">claimer</span>;
+* or select one from this [shared doc](https://hackmd.io/c6OBNgWWR8yWJhMj7WICUA?edit);
+* or if you're in a workshop, ask another participant to send you their `RequestForAttestation` object via email/another messaging system.  
```

Comment:
Awesome, very clear and to the point. 

I think you don't need this at the end. 

"if you're in a workshop, ask another participant to send you their `RequestForAttestation` object _via email/another messaging system._ "